akamai/cache-client_test.go

```diff
@@ -49,3 +54,92 @@ func TestConstructAuthHeader(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to create authorization header")
 	test.AssertEquals(t, authHeader, expectedHeader)
 }
+
+type akamaiServer struct {
+	responseCode int
+}
+
+func (as *akamaiServer) akamaiHandler(w http.ResponseWriter, r *http.Request) {
+	var req struct {
+		Objects []string
+		Type    string
+		Action  string
+	}
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		fmt.Printf("Failed to read request body: %s\n", err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	err = json.Unmarshal(body, &req)
+	if err != nil {
+		fmt.Printf("Failed to unmarshal request body: %s\n", err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	fmt.Printf("Request: %#v\n", req)
```

Comment:
Yeah, I found it useful while debugging what was going on in terms of the client <-> server interfactions, I don't think it is 100% necessary though.